.golangci.yml

```diff
@@ -11,6 +11,24 @@ linters-settings:
       - unnamedResult
   gocyclo:
     min-complexity: 20
+  goheader:
+    values:
+      regexp:
+        copyright-header: Red Hat, Inc. and others.
```

Comment:
Will this mean that only "Red Hat, Inc. and others." could be written?

I'm not sure I understand. We could have other contributors

Reply:
Good point, we shouldn’t require this specifically (the existing header linter doesn’t check for this, it looks for the license only).

Reply:
Its a regex so its not a hard-coded line. There are files that do not include this line and yet linter does not complain. For ex. https://github.com/submariner-io/submariner/blob/devel/pkg/client/listers/submariner.io/v1/endpoint.go